cmd/cluster/cleanup_leaked_ec2.go

```diff
@@ -141,6 +141,10 @@ func (c *cleanup) RemediateOCPBUGS23174(ctx context.Context) error {
 
 	resp, err := c.awsClient.DescribeInstances(ctx, &ec2.DescribeInstancesInput{
 		Filters: []types.Filter{
+			{
+				Name:   aws.String("instance-state-name"),
+				Values: []string{"running","stopped"},
```

Comment:
Ideally we add a comment here explaining why those two states.

We don't want to terminate pending because they would just be started and might not have been picked up by the MC yet. Importantly we also don't want to count terminated instances as leaked, as it causes confusion.